nautobot/utilities/testing/views.py

```diff
@@ -295,12 +296,16 @@ def test_create_object_with_permission(self):
             }
             self.assertHttpStatus(self.client.post(**request), 302)
             self.assertEqual(initial_count + 1, self._get_queryset().count())
-            if hasattr(self.model, "last_updated"):
-                instance = self._get_queryset().order_by("last_updated").last()
+            if isinstance(self._get_queryset().first(), TreeNode):
```

Comment:
Please add an explanatory comment here about why this is needed (TreeNode not supporting order_by, IIRC?)